x-pack/plugins/security_solution/public/detections/components/callouts/blog_post_detection_engineering_callout/translations.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NEW_FEATURES_BLOG_POST_CALLOUT_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.newFeaturesBlogPostCallout.calloutTitle',
+  {
+    defaultMessage: `Discover the power of Elastic's detection capabilities and unlock its potential`,
+  }
+);
+
+export const NEW_FEATURES_BLOG_POST_CALLOUT_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.detectionEngine.newFeaturesBlogPostCallout.calloutDescription',
+  {
+    defaultMessage: 'Discover new and existing detection capabilities of Elastic Security',
```

Comment:
A shorter headline might be more catchy. Feel free to omit the exclamation if you think it's too much.
```suggestion
    defaultMessage: `Discover the power of Elastic's threat detection!`,
  }
);

export const NEW_FEATURES_BLOG_POST_CALLOUT_DESCRIPTION = i18n.translate(
  'xpack.securitySolution.detectionEngine.newFeaturesBlogPostCallout.calloutDescription',
  {
    defaultMessage: 'Learn about new and existing detection capabilities of Elastic Security.',
```

Reply:
The version above is kinda minimal, if you want something short & sweet. If you want to include more specific details, maybe something like this instead:

```suggestion
    defaultMessage: `Discover the power of Elastic's threat detection `,
  }
);

export const NEW_FEATURES_BLOG_POST_CALLOUT_DESCRIPTION = i18n.translate(
  'xpack.securitySolution.detectionEngine.newFeaturesBlogPostCallout.calloutDescription',
  {
    defaultMessage: 'Learn about new and existing detection capabilities of Elastic Security — AI-powered rules, alert suppression, manual detection runs, and much more!',
```